src/lib/dnssd/Types.h

```diff
@@ -220,13 +220,13 @@ struct CommissionNodeData
     char deviceName[kMaxDeviceNameLen + 1]                    = {};
     char pairingInstruction[kMaxPairingInstructionLen + 1]    = {};
 
-    CommissionNodeData() {}
+    DnssdNodeData() {}
```

Comment:
So the problem is: most of the fields of this struct are meaningless for an operational discovery, no?  This struct is very much commissionee-discovery specific.  So naming it general "DnssdNodeData" (but leaving the "specific to commisionable/commissioning node discovery" comment above, and having all its fields be about commissionees) does not make sense.

Also, CommonResolutionData is _also_ "dnssd node data", but it's the data that is common to all discovery types, right?

Reply:
I missed that during the first view - my thought on the PR was that this is common data, however this clearly is not, like `commissionerPasscode` is clearly not an operational discovery item.